src/store/actions/apiResponse.actions.js

```diff
@@ -73,3 +78,19 @@ export const showToaster = () => ({
 export const closeToaster = () => ({
     type: CLOSE_TOASTER,
 });
+
+export const afterCloseConfirmationModal = (status, api, apiName) => (dispatch, getState) => {
+    switch (status) {
+        case 'save': {
```

Comment:
Here we can omit the bracket:
```js
switch (status) {
  case 'save':
    return confirmedSaveEndpoint(dispatch, api);
  case 'update':
    return confirmedUpdateEndpoint(dispatch, api);
}
```

Reply:
Yes, we can, and, probably we should. But I'm trying to use braces in switch-case all the time just in order to not forget them if it will be necessary.
https://github.com/airbnb/javascript/blob/master/README.md#comparison--switch-blocks

Of course, it's not our case (at least not now), but as I told, It just for any case/for future 😃 .